### DISCUSSION TITLE
feedback from an user coming from Disassembler0/Win10-Initial-Setup-Script

### DISCUSSION THREAD

**Discussion by jmlucjav** [farag2/Sophia-Script-for-Windows#160] [General]
Hi,

I used to use https://github.com/Disassembler0/Win10-Initial-Setup-Script in my win10 automated setup. It has been archived, so I found Sophia and thought it looked promising. I have now used in my first installation, and had some feedback:

1. Is there a way to FULLY automate a setup (so answers to popup/prompts would be setup beforehand), and no uses interaction would be needed whatsoever?

2. There were some things I could do in Win10-Initial-Setup-Script that I could not find in Sophia, maybe could be a good source of improvements to look at what they supported?

3. Overall it was good. I got some errors when running it, like:

```
LeaveAlone : The term 'LeaveAlone' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is
```

But it was probably me who did something wrong when using the wrapper.

cheers

**Comment by farag2** [comment]
> thought it looked promising

Thanks 😹

> Is there a way to FULLY automate a setup (so answers to popup/prompts would be setup beforehand), and no uses interaction would be needed whatsoever?

No. I dont think it's needed. At least now. As we mentioned, users dont configure the preset file properly, so its better to show them prompts. Dont think this is a big problem.

> that I could not find in Sophia

More specifically? The Sophia Script is the way it is.

> Overall it was good. I got some errors when running it, like:

May be you )) But the string number?

**Reply by jmlucjav** [reply]
> > thought it looked promising
> 
> Thanks 😹

didn't mean to sound condescending. I meant it looked like a good alternative to Win10-Initial-Setup-Script 
> 
> > Is there a way to FULLY automate a setup (so answers to popup/prompts would be setup beforehand), and no uses interaction would be needed whatsoever?
> 
> No. I dont think it's needed. At least now. As we mentioned, users dont configure the preset file properly, so its better to show them prompts. Dont think this is a big problem.
> 
> > that I could not find in Sophia
> 
> More specifically? The Sophia Script is the way it is.

These I did not find in Sophia (maybe I missed some):


SetSoundSchemeNone            # SetSoundSchemeDefault

### Network Tweaks ###
SetCurrentNetworkPrivate        # SetCurrentNetworkPublic
DisableSMB1                   # EnableSMB1
DisableConnectionSharing      # EnableConnectionSharing
DisableRemoteAssistance         # EnableRemoteAssistance
DisableRemoteDesktop

### Service Tweaks ###
DisableAutoplay                 # EnableAutoplay
DisableAutorun                  # EnableAutorun
DisableIndexing               # EnableIndexing
EnableNTFSLongPaths             # DisableNTFSLongPaths
DisableNTFSLastAccess         # EnableNTFSLastAccess
EnableHibernation          

And a a bunch from  UI Tweaks

> 
> > Overall it was good. I got some errors when running it, like:
> 
> May be you )) But the string number?

I cannot see too much details in the logs I kept, next time I'll keep them more thoroughly

thanks!

**Reply by farag2** [reply]
> didn't mean to sound condescending. I meant it looked like a good alternative to Win10-Initial-Setup-Script

Our script isnt an alternative to Win10-Initial-Setup-Script. We are alone in this domain. Every project is dead (except one Russian, but it's its' own way. It is not featured on GitHub). We do want to show users how the true tweaker should be. Not to mention that we have the most advanced tweaker. Nobody came close to us а for the last year.

**Comment by farag2** [comment]
> DisableSMB1 # EnableSMB1
SetSoundSchemeNone # SetSoundSchemeDefault
DisableConnectionSharing # EnableConnectionSharing
DisableRemoteAssistance # EnableRemoteAssistance
DisableRemoteDesktop

I do think they are unnecessary. It does more harm than good. If a user knows what the `SMB1` is he can disable it by himself. And why to turn it off? SMB1 is used only in the enterprise sector. I’ll go further than that: `SMB1` is already disabled by default for many years in 10. So to turn off that was already turned off by MS? :)

> DisableAutoplay # EnableAutoplay
EnableNTFSLongPaths # DisableNTFSLongPaths
EnableHibernation

I have this ones. Search better ))

> DisableIndexing # EnableIndexing

A dud advice. I wont let users shoot theirself in the foot. We (and I) respect our users. But if someone wants to destroy and slow down his OS, no problem, but without me. I can see almost in every tweaks in the Internet how maintaners give bad advices to turn off this and these in a pursuit of an imaginary performance. Shame on them! Win 10 cannot be speed up, just to personalized.

Also if you dont know we have a Telegram group. It much more comfortable to repy there: https://t.me/sophia_chat
